google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploymentConfiguration.java

```diff
@@ -60,15 +84,10 @@ public String getGoogleUsername() {
   }
 
   @Attribute(ENVIRONMENT_ATTRIBUTE)
-  public String getEnvironment() {
+  public AppEngineEnvironment getEnvironment() {
```

Comment:
Yep, no migration is needed. Luckily, the way IJ stores enum attributes like this is by name so this transition is safe.